src/Umbraco.Cms.Api.Management/DependencyInjection/DashboardBuilderExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Umbraco.Cms.Api.Management.Services.NewsDashboard;
+using Umbraco.Cms.Core.DependencyInjection;
+
+namespace Umbraco.Cms.Api.Management.DependencyInjection;
+
+internal static class DashboardBuilderExtensions
+{
+    internal static IUmbracoBuilder AddDashboard(this IUmbracoBuilder builder)
+    {
+        builder.Services.AddTransient<INewsDashboardService, NewsDashboardService>();
```

Comment:
I think `AddSingleton` would be fine here, instead of `AddTransient`, as the component doesn't/won't hold any state.